spring-cloud-gcp-autoconfigure/src/main/resources/org/springframework/cloud/gcp/autoconfigure/logging/logback-appender.xml

```diff
@@ -5,7 +5,7 @@ Stackdriver appender logback configuration provided for import.
 -->
 
 <included>
-  <property name="projectId" value="${projectId:-${GOOGLE_CLOUD_PROJECT}}"/>
+  <property name="projectId" value="${projectId:-${SPRING_CLOUD_GCP_LOGGING_PROJECT_ID}"/>
```

Comment:
@saturnism Did you test this? I don't see where this `projectId` property is even used? It doesn't seem to be passed to `com.google.cloud.logging.logback.LoggingAppender`, let alone the fact that `com.google.cloud.logging.logback.LoggingAppender` doesn't have that property.

Reply:
good point :/ it was in the file so i assumed it would've worked :)